src/sentry/static/sentry/app/components/noProjectMessage.jsx

```diff
@@ -1,31 +1,44 @@
 import React from 'react';
 import {Flex} from 'grid-emotion';
 import styled from 'react-emotion';
+import PropTypes from 'prop-types';
 
 import {t} from 'app/locale';
 import Button from 'app/components/button';
+import PageHeading from 'app/components/pageHeading';
 import Tooltip from 'app/components/tooltip';
 import SentryTypes from 'app/sentryTypes';
-import img from '../../../images/dashboard/hair-on-fire.svg';
+import space from 'app/styles/space';
+import img from '../../images/confused-io.png';
 
 export default class EmptyState extends React.Component {
   static propTypes = {
+    /* if the user has access to any projects, we show whatever
+    children are included. Otherwise we show the message */
+    children: PropTypes.node,
     organization: SentryTypes.Organization,
+    className: PropTypes.string,
   };
 
   render() {
-    const {organization} = this.props;
+    const {children, organization, className} = this.props;
     const orgId = organization.slug;
     const canCreateProject = organization.access.includes('project:write');
     const canJoinTeam = organization.access.includes('team:read');
+    const hasProjects =
+      organization.projects.filter(p => p.isMember && p.hasAccess).length !== 0;
```

Comment:
Instead of searching through the entire array you could stop as soon as you find one project that meets the requirements

```suggestion
      organization.projects.some(p => p.isMember && p.hasAccess);
```